docs/FAQ.md

```diff
@@ -10,7 +10,7 @@ docker run --name telegraf
 	-v /etc:/hostfs/etc:ro
 	-v /proc:/hostfs/proc:ro
 	-v /sys:/hostfs/sys:ro
-	-v /var/run/utmp:/var/run/utmp:ro
+	-v /run/utmp:/var/run/utmp:ro
```

Comment:
I don't think we should change this, even though they are the same files usually the actual file checked is /var/run/utmp.

Reply:
If you bind mount `/var/run/utmp` to `/var/run/utmp`, you will only mount the _link_, you will not mount the actual volume.  Inside the docker container you will have a link from `/var/run/utmp` to `/run/utmp`, but `/run/utmp` will be _empty_ inside the container.

You could do
```
--mount 'type=bind,source=/var/run/utmp,dst=/var/run/utmp'
--mount 'type=bind,source=/run/utmp,dst=/run/utmp'
```
which would result in the container getting both the symbolic link mounted _and_ the destination of the link mounted, though this adds complexity to the command line.

Reply:
On my Debian/Gentoo/Centos systems `/var/run` is a symlink to `/run`, but `/var/run/utmp` is the actual file, and mounting this file does work.  Since `/var/run/utmp` is the file that is actually used, mounting any other file would be making an assumption about the filesystem layout on the image.

If `/var/run/utmp` itself is actually a symlink on your system, can you let me know the distro/version?

Reply:
I believe what originally happened, thinking more about it, was that `/var/run` was mounted, instead of `/var/run/utmp` as described in the `docs/FAQ`.
Ubuntu 18.04:
```
$ ls -lah /var/run/utmp 
-rw-rw-r-- 1 root utmp 1.2K Dec  5 07:58 /var/run/utmp
$ ls -lah /run/utmp
-rw-rw-r-- 1 root utmp 1.2K Dec  5 07:58 /run/utmp
```

Reply:
> If you bind mount `/var/run/utmp` to `/var/run/utmp`, you will only mount the _link_, you will not mount the actual data. Inside the docker container you will have a link from `/var/run/utmp` to `/run/utmp`, but `/run/utmp` will be _empty_ inside the container.
> 
> You could do
> 
> ```
> --mount 'type=bind,source=/var/run/utmp,dst=/var/run/utmp'
> --mount 'type=bind,source=/run/utmp,dst=/run/utmp'
> ```
> 
> which would result in the container getting both the symbolic link mounted _and_ the destination of the link mounted, though this adds complexity to the command line.

That is the good solution, but I do not see it in the docs. It has been very hard to find this solution, already explained in #5031

In [[[inputs.system]] README](https://github.com/influxdata/telegraf/blob/master/plugins/inputs/system/README.md) tells a posible neet to add user to group `utmp`, but I have not found the relationship with the problem.